pkg/kubelet/eviction/threshold_notifier_unsupported.go

```diff
@@ -18,10 +18,16 @@ limitations under the License.
 
 package eviction
 
-import "fmt"
+import "github.com/golang/glog"
 
-// NewMemCGThresholdNotifier sends notifications when a cgroup threshold
-// is crossed (in either direction) for a given cgroup attribute
-func NewMemCGThresholdNotifier(path, attribute, threshold, description string, handler thresholdNotifierHandlerFunc) (ThresholdNotifier, error) {
-	return nil, fmt.Errorf("threshold notification not supported")
+// NewCgroupNotifier creates a cgroup notifier that does nothing because cgroups do not exist on non-linux systems.
+func NewCgroupNotifier(path, attribute string, threshold int64) (CgroupNotifier, error) {
+	glog.V(5).Infof("cgroup notifications not supported")
```

Comment:
Use Errorf, not Infof.

Reply:
oh, right.  There is no such thing as `glog.V(5).Errorf`, only `glog.Errorf`.  Ill need to revert back to what I had.

Reply:
Huh, how about that.